pkg/deploy/api/v1/types.go

```diff
@@ -268,6 +268,10 @@ type DeploymentConfigSpec struct {
 	// or failing. Post strategy hooks and After actions can be used to integrate successful deployment with an action.
 	Test bool `json:"test"`
 
+	// Paused indicates that the deployment config is paused resulting in no new deployments on template
+	// changes or changes in the template caused by other triggers.
+	Paused bool `json:"paused"`
```

Comment:
If we don't want it to show up if it's false then we want "omitempty" and not a pointer. Pointer means that there are three values in play which is not correct, a dc will be either paused or unpaused. I can change it to omitempty.